policyengine_us/variables/gov/states/hi/tax/income/credits/food_excise/hi_food_excise_exemption_amount.py

```diff
@@ -0,0 +1,47 @@
+from policyengine_us.model_api import *
+
+
+class hi_food_excise_exemption_amount(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Exemption amount for the hawaii food excise credit"
```

Comment:
Similarly elsewhere. Always capitalize states. We display these labels prominently in the app.
```suggestion
    label = "Exemption amount for the Hawaii Food/Excise Tax Credit"
```